service/controller/resource/promxy/configmap/delete.go

```diff
@@ -0,0 +1,9 @@
+package configmap
+
+import (
+	"context"
+)
+
+func (r *Resource) EnsureDeleted(ctx context.Context, obj interface{}) error {
+	return nil
```

Comment:
We don't want this deleted? Maybe something like only delete if empty? Should be good enough to avoid leftovers.

Reply:
Well, i am not sure we want to delete it anyway. As per discussions with @TheoBrigitte the lifecycle of this cm should be outside of this app